docs/release_notes/october_2022.md

```diff
@@ -0,0 +1,19 @@
+# October 2022
+## Airbyte [v0.40.13](https://github.com/airbytehq/airbyte/releases/tag/v0.40.13) to [v0.40.17](https://github.com/airbytehq/airbyte/releases/tag/v0.40.17)
+
+This page includes new features and improvements to the Airbyte Cloud and Airbyte Open Source platforms. 
+
+### New features
+* Added the low-code connector builder UI to Airbyte OSS. It includes an embedded YAML editor and significantly reduces the time and complexity of building and maintaining connectors. [#17482](https://github.com/airbytehq/airbyte/pull/17482)
+* Added Datadog Real User Monitoring (RUM) support to the webapp, which helps us monitor frontend performance in Airbyte Cloud. [#17821](https://github.com/airbytehq/airbyte/pull/17821)
+* Added Nginx and Basic Auth to ensure security when using Airbyte Open Source. [#17694](https://github.com/airbytehq/airbyte/pull/17694)
+    * Now when you start the Airbyte server and go to localhost:8000, you’ll be prompted to log in before accessing your Airbyte workspace.   
+    * You should change the default username (airbyte) and password (password) before you deploy Airbyte. If you do not want a username or password, you can remove them by setting `BASIC_AUTH_USERNAME` and `BASIC_AUTH_PASSWORD` to empty values (" ") in your `.env` file. 
+    * Our [CLI](https://github.com/airbytehq/airbyte/pull/17982) and [docs](https://docs.airbyte.com/quickstart/deploy-airbyte/) have been updated to reflect this change.
```

Comment:
@Amruta-Ranade I want to add a link to the most relevant doc changed because of the addition of basic auth to OSS. Do you think this doc is a good one to link to?

Reply:
A better link: https://docs.airbyte.com/deploying-airbyte/local-deployment